package.json

```diff
@@ -86,6 +86,7 @@
         "@babel/plugin-transform-runtime": "^7.10.4",
         "@babel/preset-env": "7.10.4",
         "@babel/preset-typescript": "^7.10.4",
+        "@cypress/react": "^4.16.4",
```

Comment:
are you using cypress as a global npm module by default since we don't have it in devdependencies? i somehow didn't have cypress installed and had to install it to be able to run

Reply:
Yup. Created a PR for documentation https://github.com/PostHog/posthog.com/pull/968